Speclens

noahfranz13 opened the pull request:
I think the analysis for this project is done for now and will be finished in the spring. I was hoping to merge it now so I can provide a link to the code in my report.

moustakas committed:
start speclens project

Commit:
update fastspecfit-speclens

Commit:
file to activate the environment

Commit:
fix output path

Commit:
start analysis

Commit:
continue analysis

Commit:
add script I used to crossmatch with masterlens

Commit:
update analysis

Commit:
add chi2 plot

Commit:
put everything in an analysis pipeline

moustakas commented:
This looks OK, although there's already a top-level `.gitignore`; why was another one needed? In addition, several of the committed files are to set up the environment; are any of these files obsolete or duplicates? Finally note that the `emlines.ecsv` file can be read from `https://github.com/desihub/fastspecfit/blob/main/py/fastspecfit/data/emlines.ecsv` but it's fine that it's checked in.

moustakas commented:
Feel free to self-merge when ready.

Commit:
add readme

Commit:
remove duplicate code

Commit:
remove more duplicate code

noahfranz13 commented:
You're right, there were two files that were unnecessary so I removed both. I never added a .gitignore to this directory, just modified the top-level .gitignore to ignore some larger files I had in my working directory. I also added a readme so that I can remember what each file does when I revisit (or anyone else visits) this repo.
